README.rst

```diff
@@ -59,54 +65,30 @@ With `use-package`
     :init
     (elpy-enable))
 
-Or if you want to defer Elpy loading:
-
-.. code-block:: elisp
-
-  (use-package elpy
-    :ensure t
-    :defer t
-    :init
-    (advice-add 'python-mode :before 'elpy-enable))
```

Comment:
Why was this deferred loading example dropped?

Reply:
I though it was not important enough to be in the readme.
It is in the doc though ([here](https://github.com/galaunay/elpy/blob/Reformat-documentation/docs/introduction.rst#with-use-package)).